apps/wallet-mobile/src/yoroi-wallets/cardano/processTransactions/processTransactions.ts

```diff
@@ -268,6 +283,6 @@ export const processTxHistoryData = (
     tokens,
     blockNumber: tx.blockNum ?? 0,
     memo,
-    metadata: metadata ?? null,
+    metadata: metadata || {},
```

Comment:
it looks is not needed if it is `??` 
```suggestion
    metadata,
```